apps/researcher/src/app/[locale]/communities/[slug]/page.tsx

```diff
@@ -162,6 +166,8 @@ export default async function CommunityPage({params}: Props) {
                 name={community.name}
                 slug={community.slug!}
                 attributionId={community.attributionId}
+                licence={community.licence}
+                licenceToAccept={env['COMMUNITY_LICENCE'] || ''}
```

Comment:
What happens if the env var does not exist?

Reply:
Good point; this should be an error because we do not want to save an empty string silently for the community.